backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -441,7 +441,7 @@ private TestResultUpload saveSubmissionToDb(
               response.getWarnings(),
               response.getErrors());
 
-      TestResultUpload finalResult = (result.getReportId() != null) ? _repo.save(result) : null;
+      TestResultUpload finalResult = _repo.save(result);
```

Comment:
can you reassign result here, or return finalResult now that we're always saving

Reply:
also can we keep this PR for db changes only, and have a follow up PR for the backend changes

Reply:
+1 on returning `finalResult`. Any particular concerns with separating into two PRs?

Reply:
separating DB changes and backend changes guarantees that the DB changes are compatible with the old backend. That comes very handy when having to revert a backend commit, you dont have to worry about reverting the db migration too.